R/processing.R

```diff
@@ -16,11 +16,16 @@ get_description <- function(pkg_folder) {
 
 #' @importFrom jsonlite toJSON
 #' @export
-parse_description <- function(pkg_folder, repo_type) {
+parse_description <- function(pkg_folder, pkg_url, repo_type) {
   message("Parsing DESCRIPTION file ...")
   description <- get_description(pkg_folder)
   description$repoType <- repo_type
+  description$tarballUrl <- pkg_url
 
+  if(!is.null(description$`Authors@R`)) {
+    authors <- eval(parse(text=description$`Authors@R`))
+    description$jsonAuthors <- authors %>% map(formatAuthor)
```

Comment:
You're sure this `map()` function is imported? If it's from `purrr`, use `@importFrom purrr map`.